spec/features/lists_all_peeps_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+
+feature "User sees the list of peeps" do
+
+  before(:each) do
+    Peep.create(content: 'Peep Peep', creation_date: Time.now)
+  end
+
+  scenario "When visiting the home page" do
```

Comment:
this name isn't very expressive. Maybe you should say 'see the peeps on the homepage'